plugin/src/main/java/io/snyk/languageserver/SnykStreamConnectionProvider.java

```diff
@@ -59,4 +60,17 @@ protected ProcessBuilder createProcessBuilder() {
     runtimeEnvironment.updateEnvironment(pb.environment());
     return pb;
   }
+
+  @Override
+  public Object getInitializationOptions(URI rootUri) {
```

Comment:
Can we have a test for this?

Reply:
Hey, so I think since this method is just delegating, the valuable test here would be an integration test at the LSP layer.

I am not too clear on how to do that, I have asked in the lsp4e GH, if you have some ideas, happy to listen.

Reply:
Hm... a unit test would be fine I guess (could also be on LsConfigurationUpdater) just checking that we fill the settings object correctly.

Reply:
Saw that you added it to LsConfigurationUpdater... resolving, thus.